openstack/networking/v3/security/groups/requests.go

```diff
@@ -0,0 +1,171 @@
+package groups
+
+import (
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/pagination"
+)
+
+// CreateOpts is a struct which will be used to create a new security group.
+type CreateOpts struct {
+	// Specifies the security group objects.
+	SecurityGroup SecurityGroupCreateOption `json:"security_group" required:"true"`
+}
+
+// SecurityGroupCreateOption is an object which will be build up the creation parameters.
+type SecurityGroupCreateOption struct {
+	// Specifies the security group name. The value can contain 1 to 64 characters, including letters, digits,
+	// underscores (_), hyphens (-), and periods (.).
+	Name string `json:"name" required:"true"`
+	// Specifies the resource ID of the VPC to which the security group belongs.
+	// Note: This parameter has been discarded, it is not recommended to use it.
+	VpcId string `json:"vpc_id,omitempty"`
+	// Enterprise project ID. The default value is 0.
+	EnterpriseProjectId string `json:"enterprise_project_id,omitempty"`
+}
+
+// Create is a method to create a new security group.
+func Create(c *golangsdk.ServiceClient, opts CreateOpts) (*SecurityGroup, error) {
+	b, err := golangsdk.BuildRequestBody(opts, "")
```

Comment:
suggest to ues `golangsdk.BuildRequestBody(opts, "security_group")`, then we can use SecurityGroupCreateOption directly.

Reply:
yeah, the struct `SecurityGroupCreateOption` and other related structs has been removed.